src/views/studio/EditVideoSheet/EditVideoSheet.tsx

```diff
@@ -203,18 +198,24 @@ export const EditVideoSheet: React.FC = () => {
     }
 
     const refetchDataAndCacheAssets = async (videoId: VideoId) => {
-      const fetchedVideo = await refetchVideo({ where: { id: videoId } })
-
+      const fetchedVideo = await client.query<GetVideosConnectionQuery, GetVideosConnectionQueryVariables>({
+        query: GetVideosConnectionDocument,
+        variables: {
+          where: {
+            channelId_eq: activeChannelId,
+            isPublic_eq: data.isPublic,
+            id_eq: videoId,
+          },
+        },
```

Comment:
We will need `orderBy` here to make sure we get the same cursor. Also, do we need `channelId_eq` and `isPublic_eq` here?